xonsh/pygments_cache.py

```diff
@@ -266,6 +267,19 @@ def cache_filename():
         )
 
 
+def add_custom_style(name, style):
```

Comment:
Can you please PR changes to this file into https://github.com/xonsh/pygments-cache.  I can cut a release for that prettu quick.

Reply:
Sure, done.

Reply:
Awesome thanks!